src/Providers/CloudWatchServiceProvider.php

```diff
@@ -67,16 +68,16 @@ public function getLogger()
      */
     public function register()
     {
-		$this->mergeConfigFrom(
-			__DIR__ . '/../../config/logging.php',
-			'logging.channels'
-		);
-
-		if (!$this->app->make('config')->get('logging.channels.cloudwatch.disable')) {
-			$this->app->singleton('cloudwatch.logger', function () {
-				return $this->getLogger();
-			});
-		}
+        $this->mergeConfigFrom(
+            __DIR__.'/../../config/logging.php',
+            'logging.channels'
+        );
+
+        if (!$this->app->make('config')->get('logging.channels.cloudwatch.disable')) {
```

Comment:
In case you wanted to change `logging.channels.cloudwatch.disable` to `logging.channels.cloudwatch.disabled`, here is another instance.

Reply:
good catch. I have fixed and pushed but the strange thing is it worked as expected. fixed anyway

Reply:
Yes, it did disable logging, but unnecessarily created the logger singleton - which by it's own wouldn't do anything either.